prow/bugzilla/client_test.go

```diff
@@ -84,6 +86,10 @@ func TestGetBug(t *testing.T) {
 		} else {
 			if id == 1705243 {
 				w.Write(bugData)
+			} else if id == 2 {
+				w.Write(bugAccessDenied)
+			} else if id == 3 {
```

Comment:
I think the `2` and `3` (and possibly the valid one too) should be named constants (`deniedBugId`, `invalidBugId`) shared between here and the callsite to avoid fragility and the need to comment there.

Reply:
It's test data, so IMHO there is no utility in investing in "DRY" to make it less "fragile". Refactors here should be intentional.